arches/app/views/notifications.py

```diff
@@ -12,7 +11,7 @@ class NotificationView(View):
     def get(self, request):
         if request.user.is_authenticated:
             if self.action == "get_types":
-                default_types = list(models.NotificationType.objects.all())
+                default_types = models.NotificationType.objects.all()
```

Comment:
This change causes an error because .index, which is a list method and not a queryset method, is used to find the matching notification_types between the user and default querysets: ```i = default_types.index(user_type.notiftype)```. It seems like the logic in this if block could be refactored a bit. I'm not sure if it's worth it though. There are never very many notification types and I doubt this `get` is called very frequently, so any improvement in performance would probably be pretty small.

Reply:
Oh good catch, I definitely got carried away. I was seeing that this was called on every page load in every base template, but I think you're right it's not worth it to understand and refactor this for one really tiny query.